aut_docs/Design_Document.md

```diff
@@ -47,3 +47,35 @@ The QA process is divided as follows:
 
 - To follow up changes in UI with changes in tests, the modifications need to be made only
   in the relevant locators/urls/page file.
+
+- Addition of any new view implies that a pom `page` and `locator` corresponding to that
+  view needs to be created.
+
+- Similarly, modifying any view implies that the corresponding `page` and `locator` will
+  also need to be modified.
+  
+## Important points regarding creation of new Test Class:
+
+- As mentioned earlier, each Test Class corresponds to a view and which has a corresponding
+  pom page and locator.
+
+- When you are creating a new Test Class, first identify the view you are creating the Class for
```

Comment:
Do not address a person using pronouns. "When creating a new test class, first identify the view the class is being created for, and create corresponding the pom page and locator."

Again, be concise. Also, don't capitalize words randomly (like ...creating the Class... here)

Reply:
Done :+1: